test/cypress/integration/widgets/deployment_button_spec.ts

```diff
@@ -293,7 +289,8 @@ describe('Create Deployment Button widget', () => {
                 cy.openAccordionSection('Advanced');
                 cy.getField('Deployment ID').find('input').clear();
                 cy.openAccordionSection('Deployment Inputs');
-                cy.clickButton('Deploy');
+                cy.contains('.dropdown', 'Install').click();
+                cy.contains('.dropdown span', 'Deploy').click();
```

Comment:
I haven't tested it but I suspect referencing `.dropdown` twice could be dropped and shortened to:
```suggestion
                cy.contains('.dropdown', 'Install').click().contains('span', 'Deploy').click();
```
or even:
```suggestion
                cy.contains('.dropdown', 'Install').click().contains('Deploy').click();
```

Reply:
Unfortunately didn't work:
![Screen Shot 2021-12-29 at 11 22 01](https://user-images.githubusercontent.com/17104212/147646755-9ea11780-3857-4520-9306-ece7177ad8b8.png)
![Screen Shot 2021-12-29 at 11 17 58](https://user-images.githubusercontent.com/17104212/147646540-1e4b1f41-0f3d-4eb8-8c37-5b33c8dba7df.png)

Reply:
I can see you repeated the reference to `.dropdown` in the second call to `contains` in L102

Reply:
I tried your second suggestion, and it worked!

Reply:
Ok, cool! 🙂

Reply:
I changed as you suggested (4f4fe59)

Reply:
I wonder how to call this function if I apply it globally. `clickDropdownDeployInDeployModal`?

Reply:
I didn't mean creating a global custom command for this as it's something specific to just deployment modal only. What I meant is just applying the refactoring in all the places where it can be applied.

Reply:
I applied it everywhere.